api/admin/routes.py

```diff
@@ -295,8 +308,7 @@ def admin_view(collection=None, book=None, **kwargs):
     setting_up = (app.manager.admin_sign_in_controller.auth == None)
     if not setting_up:
         admin = app.manager.admin_sign_in_controller.authenticated_admin_from_request()
-        csrf_token = app.manager.admin_sign_in_controller.get_csrf_token()
-        if isinstance(admin, ProblemDetail) or csrf_token is None or isinstance(csrf_token, ProblemDetail):
```

Comment:
It looks like these extra checks were removed (and not added back below) because there's no way `generate_csrf_token()` or `get_csrf_token()` can return None or a ProblemDetail. Is that right?